### PR TITLE
Format `continue` statement

### PR DESCRIPTION
**Pull request by cnpryer**
## Summary

Format `continue` statement.

## Test Plan

`continue` is used already in some tests, but if a new test is needed I could add it.

### CODE REVIEW

File: crates/ruff_python_formatter/src/snapshots/ruff_python_formatter__tests__black_test__comments2_py.snap [hunk 1]
```diff
@@ -356,10 +356,8 @@ instruction()#comment with bad spacing
 +    lcomp3 = [i for i in []]
      while True:
          if False:
--            continue
-+            NOT_YET_IMPLEMENTED_StmtContinue
- 
-             # and round and round we go
+             continue
+@@ -141,25 +104,13 @@
```
Comment (cnpryer) [inline crates/ruff_python_formatter/src/snapshots/ruff_python_formatter__tests__black_test__comments2_py.snap:360]: The ruff and black output both spit out:

```py
while True:
    if False:
        continue

        # and round and round we go
    # and round and round we go
```

cc: https://discord.com/channels/1039017663004942429/1073384953741574217/1119620838161928342

I wanna wrap my head around this, but I originally read this as "the diff excludes the comment"

Comment (cnpryer) [reply crates/ruff_python_formatter/src/snapshots/ruff_python_formatter__tests__black_test__comments2_py.snap:360]: Makes sense. "This is no longer in the diff" not "the output excludes this".

**Review by charliermarsh — APPROVED** [review]

File: crates/ruff_python_formatter/src/snapshots/ruff_python_formatter__tests__black_test__comments2_py.snap (hunk 1 above)
Comment (konstin) [reply crates/ruff_python_formatter/src/snapshots/ruff_python_formatter__tests__black_test__comments2_py.snap:360]: yeah the diff of diffs is confusing unfortunately

File: crates/ruff_python_formatter/src/statement/stmt_continue.rs [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
+use ruff_formatter::{Format, FormatResult};
 use rustpython_parser::ast::StmtContinue;
 
 #[derive(Default)]
 pub struct FormatStmtContinue;
 
 impl FormatNodeRule<StmtContinue> for FormatStmtContinue {
-    fn fmt_fields(&self, item: &StmtContinue, f: &mut PyFormatter) -> FormatResult<()> {
-        write!(f, [not_yet_implemented(item)])
+    fn fmt_fields(&self, _tem: &StmtContinue, f: &mut PyFormatter) -> FormatResult<()> {
```
Comment (konstin) [inline crates/ruff_python_formatter/src/statement/stmt_continue.rs:10]:
```suggestion
    fn fmt_fields(&self, _item: &StmtContinue, f: &mut PyFormatter) -> FormatResult<()> {
```

**Review by konstin — APPROVED** [review]

File: crates/ruff_python_formatter/src/statement/stmt_continue.rs (hunk 2 above)
Comment (cnpryer) [reply crates/ruff_python_formatter/src/statement/stmt_continue.rs]: Oof

**Pull request merged**